packages/cardano-services/src/Projection/prepareTypeormProjection.ts

```diff
@@ -347,7 +382,9 @@ export const prepareTypeormProjection = (
     entities: selectedEntities,
     extensions,
     mappers: selectedMappers,
-    stores: selectedStores
+    stores: selectedStores,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    willStore: <T>(evt: T) => willStoreCheckers.some((check) => check(evt as any))
```

Comment:
Some typeorm store operators rely on postgresql 'on delete cascade' to perform the rollback changes (delete stuff), therefore we need to process such events
```suggestion
    willStore: <T extends ProjectionEvent>(evt: T) => evt.type === ChainSyncEventType.RollBackward || willStoreCheckers.some((check) => check(evt as any))
```

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1200/commits/b16dab93e2ae49680cd1d6ef47b76b5823ce0cd0